tsconfig.module.json

```
@@ -2,7 +2,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "build": true,
+    // "build": true,
```

This previously caused an issue, but I don't think it is anymore.
```suggestion
    "build": true,
```

---

This is still causing issues: `Option '--build' must be the first command line argument`